src/Components/VerticalSplitter.lua

```diff
@@ -0,0 +1,135 @@
+local Hoarcekat = script:FindFirstAncestor("Hoarcekat")
+local Plugin = script:FindFirstAncestorWhichIsA("Plugin")
```

Comment:
Does this work when the plugin itself hot reloads? Or do we need to add stuff to the plugin facade?

Reply:
Passed pluginmouse down to VerticalSplitter from the plugin facade. Not sure if there's a more elegant solution here but it works.